styles/hero.css

```diff
@@ -55,6 +55,17 @@
   border-radius: 50%;
 }
 
+.btn {
+  transition: ease 0.5s;
+  cursor: pointer;
+}
+
+.btn:hover {
+  background-color: white;
+  color: #36b37e;
+  font-size: 1.7em;
+}
+
```

Comment:
You are not approved by GitHub actions because of linting configurations. Please try to fix the issues on your local machine. Here is the command

`stylelinxt "styles/**.css"`